cmd/wallet-web/src/layouts/OIDCShareLayout.vue

```diff
@@ -0,0 +1,81 @@
+<!--
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <keep-alive>
+    <component
+      :is="component"
+      class="
+        overflow-hidden
+        relative
+        z-10
+        flex-grow
+        justify-start
+        items-start
+        w-full
+        h-full
+        flex flex-col
+        bg-neutrals-softWhite
+      "
+    />
+  </keep-alive>
+</template>
+
+<script>
+import { reactive, ref, watch } from 'vue';
+import WACIShare from '@/pages/WACIShare.vue';
```

Comment:
Why do we need reference from WACIShare ?

Reply:
We don't, but more logic needs to be updated so that no errors are thrown. We need to create `OIDCShare` and `OIDCShareOverview` pages and use those to store in the `OIDCShareLayoutStore` (what is now still called `WACIShareLayoutStore`)